pkg/utils/release/controller.go

```diff
@@ -137,6 +143,26 @@ func (s *SuiteController) GetPipelineRunInNamespace(namespace, releaseName, rele
 	return nil, fmt.Errorf("couldn't find PipelineRun in managed namespace '%s' for a release '%s' in '%s' namespace", namespace, releaseName, releaseNamespace)
 }
 
+// GetPipelineRunListInNamespace returns the Release PipelineRun referencing the given release.
+func (s *SuiteController) GetPipelineRunListInNamespace(namespace, releaseName, releaseNamespace string) (*v1beta1.PipelineRunList, error) {
```

Comment:
I'm not seeing why we have to return the whole list here. When is any PipelineRun besides the first one found relevant? I am not even sure there will ever be more than on at once